src/test/resources/reference.conf

```diff
@@ -15,6 +15,7 @@ slick {
   port = 3306
   port = ${?MYSQL_PORT}
   db {
+    driver = "com.mysql.cj.jdbc.Driver"
```

Comment:
Yeah. Speaking of, I couldn't find a Leo config in firecloud-develop. Is it not merged yet?

https://github.com/broadinstitute/firecloud-develop/tree/dev/configs

BTW, rawls.conf has this setting: https://github.com/broadinstitute/firecloud-develop/blob/dev/configs/rawls/rawls.conf.ctmpl#L78. (Apparently `com.mysql.jdbc.Driver` is deprecated now in favor of `com.mysql.cj.jdbc.Driver`.)